index.html

```diff
@@ -0,0 +1,92 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Home</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div class="wrapper">
+        <nav class="main-nav">
+            <div class="left-bar">
+                <p class="time">9:41</p>
+                <p class="name">Oscar Byiringiro</p>
```

Comment:
Here your name represents the logo of the website, so kindly put it inside of the `a` tag instead of `p` because of three reasons:  convention, semantics, and user experience(UX). [Check this](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more info 😉